Gemfile.lock

```diff
@@ -208,15 +208,15 @@ GEM
       multi_json (~> 1.11)
       os (~> 0.9)
       signet (~> 0.7)
-    grpc (1.10.0)
+    grpc (1.12.0)
```

Comment:
We're having some issues with newer versions of `grpc` (related to our Perm integration) and we may end up needing to pin it back to `1.7.0`. Does the `cf-copilot` gem actually depend on this version or do you think it would work just as well with that older one?

Reply:
Additional context (hope to be able to link a grpc issue soon): https://www.pivotaltracker.com/n/projects/2107507/stories/157681842

Reply:
Spoke with @aaronshurley on Slack, should be fine for us to downgrade `grpc` if we need to later. Going to merge this as is.